config.sh

```diff
@@ -2,7 +2,7 @@
 function build_wheel {
     build_libs
     export ONNX_ML=1
-    time ONNX_NAMESPACE=ONNX_NAMESPACE build_bdist_wheel $@
+    time ONNX_NAMESPACE=ONNX_REL_1_3 build_bdist_wheel $@
```

Comment:
Why use a custom ONNX_NAMESPACE (instead of the standard one "onnx")?

Reply:
We in tests found that if using default namespace there could sometimes be import issue (on Linux especially) when there were other version of onnx symbols exposed. Since the namespace is customizable, we can explicitly give our released binary a unique namespace to avoid the situation.